editor/editor_properties.cpp

```diff
@@ -825,6 +825,8 @@ void EditorPropertyInteger::setup(int p_min, int p_max, bool p_allow_greater, bo
 	spin->set_step(1);
 	spin->set_allow_greater(p_allow_greater);
 	spin->set_allow_lesser(p_allow_lesser);
+	spin->set_allow_greater(true);
```

Comment:
I'm not sure this is needed, you're overriding what the two lines above do.

Reply:
Eeh... yeah that was an accident...